tests/core_test.py

```diff
@@ -142,3 +152,28 @@ def test_cyclic_peptide(input_pdb, expected_pqr, tmp_path):
         tmp_path=tmp_path,
         compare_resnames=True,
     )
+
+
+@pytest.mark.parametrize("naming_test", NAMING_TESTS, ids=str)
+def test_ph_naming(naming_test, tmp_path):
+    """Non-regression tests on naming schemes at different pH values."""
+    args = "--log-level=INFO --ff=AMBER --drop-water --apbs-input=apbs.in"
+    input_pdb = naming_test["pdb"] + ".pdb"
+    output_pqr = Path(input_pdb).stem + ".pqr"
+    expected_pqr = (
+        f"{naming_test['pdb']}_pH{naming_test['pH']}_{naming_test['ff']}.pqr"
+    )
+    args = (
+        f"--log-level=INFO --ff={naming_test['ff']} --ffout={naming_test['ff']} "
+        f"--drop-water --whitespace --with-ph={naming_test['pH']} "
+        f"--titration-state-method=propka"
+    )
+    print(args, input_pdb, output_pqr)
```

Comment:
Do you want to keep explicit prints in the test output?

Reply:
Nope -- thanks!